locales/fr-CA.js

```diff
@@ -0,0 +1,8 @@
+export const val = {
+	'buttonCancel': 'Annuler',
+	'buttonUpdate': 'Mettre à jour',
+	'inputLabelPoints': 'Points',
+	'mainBodyDescription': 'La mise à jour des détails de ces questions sera appliquée au questionnaire dans Brightspace, mais n'aura pas d'incidence sur la question d'origine créée dans l'outil d'apprentissage externe.',
+	'mainBodyWarning': 'Remarque : Pour tout questionnaire de Brightspace comportant des questions sur l'outil d'apprentissage externe, les apprenants verront une question par page.',
```

Comment:
Single quotes need to be escaped
```suggestion
	'mainBodyDescription': 'La mise à jour des détails de ces questions sera appliquée au questionnaire dans Brightspace, mais n\'aura pas d\'incidence sur la question d\'origine créée dans l\'outil d\'apprentissage externe.',
	'mainBodyWarning': 'Remarque : Pour tout questionnaire de Brightspace comportant des questions sur l\'outil d\'apprentissage externe, les apprenants verront une question par page.',
```